active/Win32/Window_and_Application_API_Spec.md

```diff
@@ -0,0 +1,655 @@
+In a Desktop app this is a wrapper for 
+[SetWindowText](https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-setwindowtexta).
+
+## Window.Current property
+_Behavior update to existing API when running as Desktop app_
+
+Gets the window of the current thread.
+
+### Remarks
+
+The value of this property depends on the thread from which it is called. 
+If called from a UI thread of a UWP app, the value is the Window instance for that thread. 
+On any other thread, the value is null.
+
+## Window.CoreWindow property
+_Behavior update to existing API when running as Desktop app_
+
+Gets the [CoreWindow](https://docs.microsoft.com/uwp/api/Windows.UI.Core.CoreWindow) associated 
+with this Window. In a Desktop app this is always null.
+
+```CS
+public CoreWindow CoreWindow { get; }
+```
+
+## Window.Dispatcher property
+_Behavior update to existing API when running as Desktop app_
+
+Gets the [CoreDispatcher](https://docs.microsoft.com/uwp/api/Windows.UI.Core.CoreDispatcher) 
+object for the Window when called from a UWP app. Returns null when called from a Desktop app.
+
+```CS
+public CoreDispatcher Dispatcher { get; }
+```
+
+The preferred alternative to this property is the `Window.DispatcherQueue` property,
+which provides a 
+[DispatcherQueue](https://docs.microsoft.com/uwp/api/Windows.System.DispatcherQueue),
+which similarly allows you to run code on the Window's UI thread.
+
+Most members of the Window class can only be accessed when running on the thread the object was
+created on. This property, though, can be called from any thread.
+
+## Window.DispatcherQueue property
+
+Gets the [DispatcherQueue](https://docs.microsoft.com/uwp/api/Windows.System.DispatcherQueue) 
+object for the Window.
+
+```CS
+public DispatcherQueue DispatcherQueue { get; }
+```
+
+Most members of the Window class can only be accessed when running on the thread the object was
+created on. This property, though, can be called from any thread.
+
+> Spec note: This Window.DispatcherQueue property matches the new DependencyObject.DispatcherQueue property
+[described here](https://github.com/microsoft/microsoft-ui-xaml-specs/blob/master/winui3/DispatcherQueueUpdates.md)
+
+
+## IWindowNative COM Interface
+
+This interface is implemented by `Window`, and in a Desktop app can be used to get 
+the Window's underlying HWND.
+
+```CS
+protected override void OnLaunched(Microsoft.UI.Xaml.XamlLaunchActivatedEventArgs e)
+{
+   m_window = new MainWindow();
+   m_window.Activate();
+   
+   // AS is an extension method in the WinRT.CastExtensions class. 
+   // The WinRT namespace is needed to use the extension method.
+   var windowNative = m_window.As<IWindowNative>();
+   HWND hwnd = windowNative.WindowHandle;
+   ...
+}
+```
+
+## Application class
+
+> Spec note: Application is not new to this spec, what is new are the following remarks
+
+### Remarks
```

Comment:
Should we do some refactoring here, since it gets confusing if we have a lot of things that never work for Win32, and this is becoming a front and center scenario?